src/main/pages/user-management/organizations.adoc

```diff
@@ -65,76 +82,114 @@ CHE_MAIL_SMTP_AUTH_PASSWORD=password
+You can only add members of the parent organization as members of the sub-organization.
 
 [id="organization-and-sub-organization-administration"]
-== Organization and Sub-Organization Administration
+== Organization and sub-organization administration
 
-Organization settings are visible to all members of the organization, but only the Eclipse Che system administrator is able to modify the settings.
+The settings of the organization are visible to all members of the organization.  Only the Eclipse Che system administrator modifies the settings.
 
 [id="rename-an-organization-or-sub-organization"]
-== Rename an Organization or Sub-Organization
+== Renaming an organization or sub-organization
 
-*Action restricted to*: Eclipse Che system administrator and admins of the organization.
+[NOTE]
+====
+Only an Eclipse Che system administrator and administrator of the organization can rename an organization or sub-organization.
+====
 
-To rename an Organization, click in the "Name" textfield and start editing the name of the organization. Once edited, the save mode will appear - click on "Save" button to update the name.
+To rename an organization:
+
+. Click *Name* field to edit the name of the organization.  The save mode displays after typing.
+
+. Click *Save* button to update the name.
 
 The name of the organization is restricted to the following rules: +
-- Only alphanumeric characters or a single "-" can be used +
-- Spaces cannot be used in organization names +
-- Each organization name must be unique within the Eclipse Che install - Each sub-organization name must be unique within an organization
+* Only alphanumeric characters and a single "-" can be used. +
+* Spaces can not be used in the name of the organization. +
+* Each organization name must be unique within the Eclipse Che install. +
+* Each sub-organization name must be unique within an organization.
 
 [id="leave-an-organization-or-sub-organization"]
-== Leave an Organization or Sub-Organization
+== Leaving an organization or sub-organization
 
-This action is not possible for members of an organization. Users have to contact organization’s admin or Eclipse Che system admin.
+Members of an organization can not leave an organization or sub-organization. To leave the organization, contact the administrator of the organization or the system administrator of Eclipse Che.  
 
 [id="delete-an-organization-or-sub-organization"]
-== Delete an Organization or Sub-Organization
+== Deleting an organization or sub-organization
 
-*Action restricted to*: Eclipse Che system administrator and admins of the organization.
+[IMPORTANT]
+====
+* This action can not be reverted and all workspaces created under the organization will be deleted.
+* All members of the organization will receive an email notification to inform about the deletion of the organization.
+* Only system adminstrators or administrators of the organization delete an organization or sub-organization.
```

Comment:
I have a question about the use of "can".  I thought that "may" and "can" should be avoided.  Or maybe it's just "may".

Reply:
It's just "may" because it (usually) denotes uncertainty. In this case, we want to say that only admins have the capability to do something. So it's fine to use "can". You could also say "have the rights" or something to that effect.